org-kanban.el

```diff
@@ -103,14 +103,40 @@
      (define-key map org-kanban/next-key (lambda () (interactive) (org-kanban/shift 'right)))
      map)))
 
-(defun org-kanban/initialize ()
+(defun org-kanban/initialize (&optional arg)
   "Create an org-kanban dynamic block"
+  (interactive "p")
+  (message "%s" arg)
+  (cond ((eq arg nil) (org-kanban/initialize-here))
+        ((eq arg 1) (org-kanban/initialize-here))
+        ((eq arg 4) (org-kanban/initialize-at-beginning))
+        ((eq arg 16) (org-kanban/initialize-at-end))
+        (t nil)))
```

Comment:
can we remove this or throw an error?
should not happen anyways and is not supported ...